css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
```

Comment:
Please use 2 spaces for tabs, see the Google style guide for more info: https://google.github.io/styleguide/htmlcssguide.html#Indentation.